docs/writing-docs/tutorials/resources.md

```diff
@@ -79,11 +79,69 @@ controller.A.onEvent(ControllerButtonEvent.Pressed, function () {
 ```
 ````
 
+## assetsjson
 
-## Custom assets (Arcade beta only)
+If you want a collection of pre-created tilemaps, images, animations, and tiles included as `assets`
+in a tutorial, you can add them inside an ```` ```assetsjson ```` block. This makes the assets
+available in the **Assets Editor**, and to the code in the tutorial, when the tutorial is started.
 
-To import all assets (Tilemaps, Images, Animations, and Tiles) into the tutorial at once, use the ```` ```assetsjson ``` ```` language block. You can get the assets for your project by adding `?savetemplate=1` to the end of your URL. For example, the URL for arcade's beta editor would be:
+### ~reminder
+
+#### Asset editing support
+
+Support for creating and importing assets is only available in [Microsoft MakeCode Arcade](https://arcade.makecode.com).
+
+### ~
+
+### Importing custom assets
+
+To import all assets (tilemaps, images, animations, and tiles) into the tutorial at once,
+start the editor by adding `?savetemplate=1` to the end of your URL. For example, the URL
+for Arcade's beta editor would be:
 
 https://arcade.makecode.com/beta?savetemplate=1
 
-With the correct URL entered, you can click the save button (the floppy disk) in the editor to download your project as a `.txt.mkcd` file. The contents of this file can be opened in a text editor and pasted into the ```` ```assetsjson ``` ```` block to configure the tutorial. To reopen and edit the project, simply drag the original `.txt.mkcd` into the editor and repeat the process to download it again.
\ No newline at end of file
+With the correct URL entered, you can click the save button (the floppy disk button) in the
+editor to download your project as a `.txt.mkcd` file.
+
+Next, define a ```` ```assetsjson ```` block in your tutorial document.
```

Comment:
ah! this should be singular `assetjson`, the previous docs were wrong

Reply:
Eeek! Ok.

Reply:
thanks galen!!